src/main/java/seedu/address/logic/commands/ClientEditCommand.java

```diff
@@ -86,6 +88,7 @@ public CommandResult execute(Model model) throws CommandException {
 
         model.setClient(clientToEdit, editedClient);
         model.updateFilteredClientList(PREDICATE_SHOW_ALL_CLIENTS);
+        model.initialiseTagNoteMap();
```

Comment:
should this be here?

Reply:
removed, was unnecessary init. The `setClient` method covers this syncing of the tagNoteMap now